### ISSUE TITLE
Text erroneously deleted when using editor on iPhone with ENTER_BR option set

### ISSUE THREAD

**Issue by burtonrhodes** [froala/wysiwyg-editor#4757]
##### Expected behavior.
Pressing enter within the browser should only create a new line.

##### Actual behavior.
When pressing enter (twice) in iPhone browsers (tested in Safari and Chrome) when using Froala option "enter: ENTER_BR", the text below is deleted.

##### Steps to reproduce the problem.

1. Must be on iPhone (or iPhone simulator) using Safari or Chrome 
2. Place cursor on empty line with text below Type any text (e.g. "Test") and press ENTER twice
3. The text below is deleted

##### Editor version.
Froala v4.1.4

##### OS.
iOS 17.4 (also was happening on previous versions)

##### Browser.
Bug reproducible on iOS Safari and iOS Chrome

##### Recording.
https://www.loom.com/share/81c8a12d88cd46d3861ec043035aa620?sid=fdf2f906-129b-498d-80ee-4017d1b84323

##### jsFiddle.
https://jsfiddle.net/burtonrhodes/6zqeLrcs/

##### Embedded jsFiddle (to test with iPhone or iPhone simulator)
https://www.aframesoftware.com/froala_test/

**Comment by RevaJS** [comment]
This has been happening to me for almost 2 years now. It would be wonderful to actually be able to use this feature!

**Comment by MorganKing23** [comment]
Yes, I would love for this to be fixed so we can easily write emails from our phones!
